parquet-column/src/main/java/parquet/schema/Types.java

```diff
@@ -555,6 +607,342 @@ protected GroupType build(String name) {
           "Cannot build an empty group");
       return new GroupType(repetition, name, originalType, fields, id);
     }
+
+    public MapBuilder<Q> map(
+        Type.Repetition repetition) {
+      return new MapBuilder<Q>(self()).repetition(repetition);
+    }
+
+    public MapBuilder<Q> requiredMap() {
+      return new MapBuilder<Q>(self())
+          .repetition(Type.Repetition.REQUIRED);
+    }
+
+    public MapBuilder<Q> optionalMap() {
+      return new MapBuilder<Q>(self())
+          .repetition(Type.Repetition.OPTIONAL);
+    }
+
+    public MapBuilder<Q> repeatedMap() {
```

Comment:
We decided in PARQUET-113 to not allow both MAP / LIST annotations and unannotated repeated groups. So we don't need `repeatedMap` or `repeatedList`. :)